matlab_simulation/code v1.0/ME597-5-ExtendedKalmanFilter/omniRobotFrame.m~

```diff
@@ -0,0 +1,11 @@
+function [Xideal,X,Y,V]=omniRobotFrame(r,l,W,X0,dt,e)
+
```

Comment:
% W IS VEKTOR OF THE WHEELS VELOCITIES W=[W1, W2, W3]
% V IS VELOCITY OF THE OMNIDIRECTIONAL ROBOT